.azure-pipelines/LinuxAnalyzerTest.yml

```diff
@@ -4,6 +4,7 @@ jobs:
     vmImage: ubuntu-18.04
   variables:
     FLUTTER_HOME: /opt/flutter
+    phpVersion: 7.2
```

Comment:
According to these docs you also have to run `update-alternatives` several times:
https://docs.microsoft.com/en-us/azure/devops/pipelines/ecosystems/php?view=azure-devops#use-a-specific-php-version